src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -22,7 +22,7 @@
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model = new ModelManager(TypicalAddressBook.getTypicalAddressBook(), new UserPrefs());
```

Comment:
Take note that we will need to further refactor and rename path, class names, variable names, and method names with AddressBook to LibTask to ensure consistency with our tP project.

For example here ```TypicalAddressBook``` to ```TypicaLibTask```.